CHANGELOG.md

```diff
@@ -7,7 +7,7 @@
   </summary>
 
 ### Minor
-
+- Icon: Add compose icon (#352)
```

Comment:
i just realized this is actually #358 not #352 , do you mind changing @jcpark89 https://github.com/pinterest/gestalt/pull/358/